autopush/metrics.py

```diff
@@ -79,59 +87,50 @@ def timing(self, name, duration, **kwargs):
         self._metric.timing(name, duration)
 
 
-def make_tags(base=None, **kwargs):
-    # type: (Sequence[str], **Any) -> Sequence[str]
-    """Generate a list of tag values"""
-    tags = list(base or [])
-    tags.extend('{}:{}'.format(key, val) for key, val in kwargs.iteritems())
-    return tags
+class TaggedMetrics(object):
+    """DataDog like tagged Metric backend"""
+    def __init__(self, hostname, namespace="autopush"):
 
-
-class DatadogMetrics(object):
-    """DataDog Metric backend"""
-    def __init__(self, api_key, app_key, hostname, flush_interval=10,
-                 namespace="autopush"):
-
-        datadog.initialize(api_key=api_key, app_key=app_key,
-                           host_name=hostname)
-        self._client = ThreadStats()
-        self._flush_interval = flush_interval
+        markus.configure(
+            backends=[{
+                'class': 'markus.backends.datadog.DatadogMetrics',
+                'options': {
+                    'statsd_host': hostname,
+                    'statsd_namespace': namespace,
+                }}])
+        self._client = markus.get_metrics(namespace)
         self._host = hostname
         self._namespace = namespace
 
     def _prefix_name(self, name):
-        return "%s.%s" % (self._namespace, name)
+        return name
+        # return "%s.%s" % (self._namespace, name)
```

Comment:
If we don't need this commented out code it can be removed (applies generally to the PR).

Reply:
Partly kept it for sanity reasons, but agree it's not super important.

Reply:
There's more commented-out code in [`test_router`](https://github.com/mozilla-services/autopush/pull/1401/files#diff-fd4bdb0e2090991d9609f72d3920c0edR609). Is that expected to be added back soon (or modified) like the skipped tests, or can it also be removed?

Reply:
Ah, no. missed that. Thanks!